src/InsertionsClient/Api/Props/SwrFileReader.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Net.Insertions.Api.Models;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Net.Insertions.Api
+{
+    /// <summary>
+    /// Finds and loads .swr files in a given directory.
+    /// </summary>
+    internal class SwrFileReader
+    {
+        private readonly int _maxConcurrency;
+
+        /// <summary>
+        /// Pattern that finds payload path in swr file content.
+        /// </summary>
+        private Regex _payloadPattern = new Regex(@"\bvs.payload.*source=(.+?)($|\s)");
```

Comment:
pointing out the heterogeneity in how class members are organized in this PR.  In this particular class, is the c-sharpie way.
- fields, .ctors, properties + indexers, methods
- instance fields with prefix `_`

Reply:
Noted. I will keep that in mind :)